core/domain/suggestion_services.py

```diff
@@ -2810,7 +2810,7 @@ def update_translation_contribution_stats_at_submission(
             accepted_translation_word_count=0,
             rejected_translations_count=0,
             rejected_translation_word_count=0,
-            contribution_dates=[suggestion.last_updated.date()]
+            contribution_dates=sorted([suggestion.last_updated.date()])
```

Comment:
Oh, sorry, I didn't realize this was updated so directly. My bad, it's actually fine to remove the sorted() from here and line 3009 -- apologies, I wasn't sure where to look before.

Otherwise the PR LGTM. Thanks!